### DISCUSSION TITLE
how to use jquery stop playing.

### DISCUSSION THREAD

**Discussion by gzshawnliang** [cifkao/html-midi-player#34] [Q&A] [answered]
Hi,i am new to web dev.The html-midi-player work well in jquery,thanks for your job.But how to use jquery stop the html-midi-player playing?

**Comment by cifkao** [comment, marked as answer]
You just need to get the actual HTML element from your jQuery object and call `stop()` on it directly. I guess something like `$("#myPlayer")[0].stop()` will work. Or if your query matches multiple elements, loop over them with `each()` and call `stop()` on each one.

**Reply by gzshawnliang** [reply]
It's worked.Thanks for your reply.
